restonomer-spark-utils/src/main/scala/com/clairvoyant/restonomer/spark/utils/transformer/DataFrameTransformerImplicits.scala

```diff
@@ -61,6 +61,33 @@ object DataFrameTransformerImplicits {
 
     def colToJson(columnName: String): DataFrame = df.withColumn(columnName, to_json(col(columnName)))
 
+    def changeColCase(caseType: String): DataFrame = {
+
+      def changeColCaseFunc(colName: String, caseType: String): String =
+        caseType match {
+          case "upper" =>
+            colName.toUpperCase
+          case "lower" =>
+            colName.toLowerCase
+          case _ =>
+            colName
```

Comment:
Instead of returning the same column name, throw an exception that the given case conversion is not supported.

Reply:
resolved